docs/csharp/toc.yml

```
@@ -56,7 +56,7 @@
     href: tutorials/upgrade-to-nullable-references.md
   - name: Generate and consume asynchronous streams
     href: tutorials/generate-consume-asynchronous-stream.md
-  - name: Extend data capabilities using pattern matching
+  - name: Conditionally return values with pattern matching
```

Hmm. I'm not thrilled with that title (although I agree it should change)
How about:

```suggestion
  - name: Build data-driven algorithms with pattern matching
```

---

Sure, sounds good to me.